scene/gui/control.h

```diff
@@ -292,8 +292,8 @@ class Control : public CanvasItem {
 	void _update_minimum_size();
 	void _size_changed();
 
-	void _toplevel_changed() override{}; // Controls don't need to do anything, only other CanvasItems.
-	void _toplevel_changed_on_parent() override;
+	void _top_level_changed() override{}; // Controls don't need to do anything, only other CanvasItems.
```

Comment:
```suggestion
	void _top_level_changed() override {}; // Controls don't need to do anything, only other CanvasItems.
```

Not sure why clang-format doesn't complain about it, but I'm pretty sure we need a space here by convention.

Reply:
The semicolon needs to be removed, that's what confuses clang-format here.

Reply:
yay me ... found a way to circumvent clang-format :-D